src/RecoveryServices/RecoveryServices.SiteRecovery/ReplicationProtectedItem/NewAzureRmRecoveryServicesAsrReplicationProtectedItem.cs

```diff
@@ -604,6 +663,12 @@ private void EnterpriseAndHyperVToAzure(EnableProtectionInput input)
             providerSettings.TargetProximityPlacementGroupId = this.RecoveryProximityPlacementGroupId;
             providerSettings.TargetAvailabilityZone = this.RecoveryAvailabilityZone;
             providerSettings.UseManagedDisks = this.UseManagedDisk;
+            providerSettings.TargetAvailabilitySetId = this.RecoveryAvailabilitySetId;
+            providerSettings.TargetVmSize = this.Size;
+            providerSettings.SqlServerLicenseType = this.SqlServerLicenseType;
+            providerSettings.TargetVmTags = this.RecoveryVmTag;
+            providerSettings.TargetManagedDiskTags = this.DiskTag;
```

Comment:
>providerSettings.TargetManagedDiskTags = this.DiskTag; [](start = 12, length = 54)

Only if useManagedDisks is true else service will throw error.
Should we throw error in powershell or we want to surface the srs exception??

Reply:
Handled in powershell

---
In reply to: [601548064](https://github.com/AsrOneSdk/azure-powershell/pull/21#discussion_r601548064) [](ancestors = 601548064)